ui/admin-portal/src/app/components/candidates/visa/visa-job-assessments/modal/create-visa-job-assessement.component.ts

```diff
@@ -55,10 +54,10 @@ export class CreateVisaJobAssessementComponent implements OnInit {
   }
 
   onSelect() {
-    const request: CreateCandidateVisaJobRequest = {
-        name: this.name,
-        sfJobLink: this.sfJoblink };
-    this.activeModal.close(request);
+    // const request: CreateCandidateVisaJobRequest = {
```

Comment:
Are these commented out lines, going to go back in - or are they gone for good? 

If you are going to be coming back to this code, there should be an issue raised - and maybe a todo comment as well.

If you are not coming back to the code, then just remove the code.